tests/Oryx.Integration.Tests/Python/PythonDynamicInstallationTest.cs

```diff
@@ -111,7 +111,8 @@ public async Task CanBuildAndRunPythonAppAsync(string pythonVersion)
         }
         [Theory]
         [InlineData(PythonVersions.Python310Version)]
-        public async Task CanBuildAndRunPythonApp_UsingGitHubActionsBullseyeBuildImage_AndDynamicRuntimeInstallationAsync(
+        [InlineData(PythonVersions.Python311Version)]
+        public async Task CanBuildAndRunPythonApp_UsingGitHubActionsBullseyeBuildImage_AndDynamicRuntimeInstallation(
```

Comment:
nit: any reason `Async` was removed from the method name?

Reply:
Updated